### PR TITLE
Cb deploy service

### PR DESCRIPTION
**Pull request by Anilreddy999**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -69,6 +69,8 @@ pipeline:
   deploy_to_dev:
     image: quay.io/ukhomeofficedigital/kd
     secrets:
+    - source: DEV_SGMR_API_URL
```
Comment (belindac-vd) [inline .drone.yml]: Use dev_sgmr_data_api_base_url instead otherwise we have multiple variables with the same value

File: .drone.yml [hunk 2]
```diff
@@ -77,6 +79,10 @@ pipeline:
       target: SGMR_SERVICE_URL
     - source: DEV_SGMR_KUBE_TOKEN
       target: KUBE_TOKEN
+    - source: ENVIRONMENT
+      target: ENVIRONMENT
+    - source: PROTOCOL_HTTPS
```
Comment (belindac-vd) [inline .drone.yml]: don't need this if you use dev_sgmr_data_api_base_url

File: env.yaml [hunk 3]
```diff
@@ -4,16 +4,20 @@ keys:
...
       aws_secret_access_key
       private_token
+  - environment
   - kube:
       namespace_cop_sgmr
       server
   - nginx:
       image
       tag
+  - protocol:
```
Comment (belindac-vd) [inline env.yaml]: don't need this

File: env.yaml [hunk 4]
```diff
@@ -4,16 +4,20 @@ keys:
...
+      https
   - quay:
       password
       username
   - sgmr:
+      api_url
```
Comment (belindac-vd) [inline env.yaml]: change to data_api_base_url

File: kube/deployment.yml [hunk 5]
```diff
@@ -69,6 +69,11 @@ spec:
           limits:
             memory: "2304Mi"
             cpu: "1000m"
+        env:
+          - name: API_BASE_URL
+            value: {{.PROTOCOL_HTTPS}}{{.SGMR_API_URL}}
```
Comment (belindac-vd) [inline kube/deployment.yml]: Use SGMR_API_URL only

File: .drone.yml (hunk 1 above)
Comment (Anilreddy999) [reply .drone.yml]: I have no idea that dev_sgmr_data_api_base_url exists. I created new URL as informed. Now, updated

File: env.yaml (hunk 3 above)
Comment (Anilreddy999) [reply env.yaml]: I have no idea that dev_sgmr_data_api_base_url exists. I created new URL as informed. Now, updated

File: env.yaml (hunk 4 above)
Comment (Anilreddy999) [reply env.yaml]: Updated now

File: kube/deployment.yml (hunk 5 above)
Comment (Anilreddy999) [reply kube/deployment.yml]: Updated now

**Pull request merged**